### PR TITLE
docs: Adds Troubleshooting Section to Serverless

### PR DESCRIPTION
**Pull request by dthyresson**
This documentation change helps resolve a deployment issue with serverless if your dev instance is running.

See:

image

### CODE REVIEW

**Comment by jtoar** [comment]
@dthyresson I can review and get this one in if it's ready for review?

**Comment by dthyresson** [comment]
> @dthyresson I can review and get this one in if it's ready for review?

Sure thing. It's just a small docs update based on a convo in Discord I noticed today.

File: docs/docs/deploy/serverless.md [hunk 1]
```diff
@@ -109,3 +109,16 @@ This will take several minutes, so grab your favorite beverage and enjoy your ne
 :::tip Pro tip
 If you get tired of typing `serverless` each time, you can use the much shorter `sls` alias: `yarn rw deploy sls`
 :::
+
+## Troubleshooting
+
+If you happend to see the following error wehn deploying:
```
Comment (jtoar) [inline docs/docs/deploy/serverless.md:115]:
```suggestion
If you happen to see the following error when deploying:
```

File: docs/docs/deploy/serverless.md [hunk 2]
```diff
@@ -109,3 +109,16 @@ This will take several minutes, so grab your favorite beverage and enjoy your ne
...
+
+```terminal
+Error:
+No auth.zip file found in the package path you provided.
+```
+
+And when you `run yarn rw deploy serverless --first-run`, these files get created, but then dissapear and that error gets thrown, please make sure that you are not running your dev instance when you attempt to deploy.
+
+Please stop your dev server and then retry your deploy.
```
Comment (jtoar) [inline docs/docs/deploy/serverless.md:124]:
```suggestion
Make sure that the dev server isn't running, then retry your deploy.
```

**Review by jtoar — COMMENTED** [review]
@dthyresson based on the screenshot you posted this seemed like enough, what do you think?

**Pull request merged**